src/main.ts

```diff
@@ -397,7 +397,7 @@ app.on('ready', () => {
 
   autoUpdater.on('error', (error) => {
     console.error('updater-error', error)
-    mainWindow?.webContents.send('updater-error', error)
```

Comment:
https://github.com/KittyCAD/modeling-app/blob/349368742c73bd32383dddbf2c490d7bc903bafb/src/preload.ts#L38-L39

This was the typo causing some updater errors to never get surfaced to the web layer.